tidb-cloud/tidb-cloud-auditing.md

```diff
@@ -189,6 +183,13 @@ For example, `13796619446086334065/tidb-0/tidb-audit-2022-04-21T18-16-29.529.log
 
 If you no longer want to audit a cluster, go to the page of the cluster, click **Settings** > **Audit Settings**, and then toggle the audit setting in the upper-right corner to **Off**.
 
+> **Note:**
+>
+> After the audit log is disabled, the last log file will not be pushed to the bucket on the cloud. You can push this file in two ways:
```

Comment:
just clarify that the log will and will only be pushed once it reaches size 10M.
how can the user restart a cluster?, scale-out not work